lib/parsers.js

```diff
@@ -49,9 +49,67 @@ function parserLinux(lines, search) {
   return ports;
 }
 
-// TODO: Windows parser (and add it below!)
+function parserWin32(lines, search) {
+  var ports = [];
+  if (!isNonEmptyArray(lines)) {
+    return [];
+  }
+
+  var tasklistLines = lines[0];
+  if (!isNonEmptyArray(tasklistLines)) {
+    return [];
+  }
+
+  var netstatLines = lines[1];
+  if (!isNonEmptyArray(netstatLines)) {
+    return [];
+  }
+
+  // Example tasklist syntax:
+  // b2g-bin.exe                  16180 RDP-Tcp#2                  1    129,560 K
+  var pidMap = {};
+  var tasklistRegex = new RegExp(
+    '^(' + search.join('|') + ')(?:-bin)?\\.exe\\s+([\\d]+)'
```

Comment:
Yeah, though the line was >90 col wide, so I thought that might be a bit more readable. I can join it up though